### DISCUSSION TITLE
add dominance plot as a diagnostic for the matching

### DISCUSSION THREAD

**Discussion by wxwx1993** [NSAPH-Software/CausalGPS#212] [Ideas]
@jennyjyounglee, developed a dominance plot tool as the diagnostic for the matching. It is useful to be an enhancement.
https://github.com/jennyjyounglee/metagam

**Comment by Naeemkh** [comment]
@wxwx1993, please add more details for this feature.

**Comment by Naeemkh** [comment]
@wxwx1993, @jennyjyounglee, please point me to the exact code.
